tests/Settings.parseArguments.phpt

```diff
@@ -103,6 +103,14 @@ class SettingsParseArgumentsTest extends Tester\TestCase
         Assert::equal(Settings::FORMAT_JSON, $settings->format);
     }
 
+    public function testGitLabOutput()
```

Comment:
This test only checks if the `--gitlab` argument works. You should also test if the JSON output is valid according to [GitLab's specifications][1] (or maybe even [CodeClimate's standards][2]).

[1]: https://docs.gitlab.com/ee/user/project/merge_requests/code_quality.html#implementing-a-custom-tool
[2]: https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#data-types

Reply:
I agree. Please, can you add the unit test? I will be easy :)

Reply:
I have added a test for this. I hope this is what you had in mind for your tests.